Python Solutions/Container With Most Water

```diff
@@ -0,0 +1,19 @@
+class Solution:
+    def maxArea(self, height: List[int]) -> int:
+        # initialize the result, which keeps track of the maximum area
+        res = 0
+        # left pointer - keeps track of left index
+        # right pointer - keeps track of the right index
+        left , right = 0, len(height)-1
+
+        while left < right:
+            area = (right - left) * min(height[left], height[right])
+            # update the maximum area
+            res = max(area, res)
+            # This was necessary because the left and right pointer moves through the height once
```

Comment:
> I don't understand this part. I think it would better to explain like why we move the left pointer when height[left] < height[right], and the right pointer when height[left] > height[right]. 
-> something like: 
```
            if height[left] < height[right]:
                # increment left to _____
                left += 1
            else:
               # decrement right to _____
                right -= 1
```

Reply:
That Sounds good, I will add a comment that explains that